docs/usage_and_configuration/ways_to_install_black.md

```diff
@@ -0,0 +1,46 @@
+# Ways to install Black
+
+There are several ways you can install _Black_.
+
+## Install from PyPI
+
+To install the latest release of _Black_ from [PyPI] (Requires Python 3.8+), use:
+
+`pip install black`
+
+A _Black_ release currently offers three types of artifacts via PyPI, as outlined in the
+[Release Process]:
+
+1. The source distribution of the release
+2. Generic Python wheel, meant for use on any Python supported platform
+3. Platform and Python version specific wheels that offer significantly improved
+   performance, compiled using [mypyc]
+
+By default, `pip` will prefer a compatible wheel and revert to the source distribution
+if no such wheels are found, as outlined in [Python documentation].
+
+## Install from GitHub
+
+To install the latest version of _Black_ from GitHub, use:
+
+`pip install git+https://github.com/psf/black`
+
+## Get native binaries from GitHub
+
+[GitHub Releases] for _Black_ contain self-contained, native binaries for multiple
+platforms (built using PyInstaller). This allows you to download the executable for your
+platform and run _Black_ without a Python runtime installed.
+
+## Black Docker images
+
+Official _Black_ Docker images are available on [Docker Hub]. For more information,
+check the [Black Docker image] section.
```

Comment:
```suggestion
Official _Black_ Docker images are available on [Docker Hub](https://hub.docker.com/r/pyfound/black). For more information,
check the [Black Docker image](LINK_TO_PAGE) page.

- `docker run --rm --volume $(pwd):/src --workdir /src pyfound/black:latest_release black --check .`
```

Command example optional but I think we should fix the link to make this page more useful.

Reply:
I left out the command example because it's already present in the Black Docker page, can add it if required.